### DISCUSSION TITLE
[Projects Beta] Board view, configurations, sort option doesn't work

### DISCUSSION THREAD

**Discussion by Shameness** [community/community#10900] [Projects and Issues] [answered]
Expected Board view to sort each column. Doesn't work

1. Create a board view
2. Create a new field: foobar
3. Set configuration: column field to foobar
4. Set configuration: sort to Status-desc

Result: Column entities are not ordered by Status field
Expected: Columns entities ordered by Status field

**Comment by Arufonsu** [comment]
It won't properly sort everything by pull requests in the board view either 😢 only few categories are re-sorted.

**Comment by rileybroughten** [comment, marked as answer]
Hello! 👋 You can sort by field values from the board layout similar to other layouts, check out our announcement [here](https://github.blog/changelog/2023-01-26-github-projects-january-26th-update/#sorting-by-field-values-on-the-board-layout)!
